### PR TITLE
Removed deprecations in Filesystem component

### PR DESCRIPTION
**Pull request by dosten**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | - |
| License | MIT |
| Doc PR | - |

### CODE REVIEW

File: src/Symfony/Component/Filesystem/Filesystem.php [hunk 1]
```diff
@@ -479,9 +477,6 @@ public function dumpFile($filename, $content, $mode = 0666)
         }
 
         $this->rename($tmpFile, $filename, true);
-        if (null !== $mode) {
-            $this->chmod($filename, $mode);
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Filesystem/Filesystem.php]: Looks like we missed triggering a deprecation notice here in 2.7?

Comment (dosten) [inline src/Symfony/Component/Filesystem/Filesystem.php]: Yes, you are right, in what branch must be applied a patch for this?

Comment (nicolas-grekas) [inline src/Symfony/Component/Filesystem/Filesystem.php]: We add these to 2.7

Comment (dosten) [inline src/Symfony/Component/Filesystem/Filesystem.php]: The default value of the `$mode` param is 0666, so the error will be triggered ever

**Comment by aitboudad** [comment]
can you also update the CHANGELOG.

**Comment by dosten** [comment]
@aitboudad sorry i don't understand, update the CHANGELOG.md for what?

**Comment by aitboudad** [comment]
@dosten just add note about the removed deprecations, you can see https://github.com/symfony/symfony/blob/master/src/Symfony/Component/Form/CHANGELOG.md for example

**Comment by dosten** [comment]
@aitboudad Done!

**Comment by dosten** [comment]
ping @stof @fabpot

File: src/Symfony/Component/Filesystem/Filesystem.php (hunk 1 above)
Comment (xabbuh) [inline src/Symfony/Component/Filesystem/Filesystem.php]: see #14460

**Comment by fabpot** [comment]
Thank you @dosten.

**Pull request closed without merging**